src/forms/EditListingLocationForm/EditListingLocationForm.css

```diff
@@ -50,7 +50,7 @@
 .predictionsRoot {
   position: absolute;
   width: 100%;
-  padding-bottom: 98px;
+  padding-bottom: var(--locationAutocompleteBottomPadding);
```

Comment:
I think the larger padding here was for the Mobile Safari bottom spacing, but it looked fine with 72px when I tested it.